### PR TITLE
Add file to populate the quickstart h2 db from scratch

### PR DESCRIPTION
**Pull request by ravivj-db**
**PR Checklist**

- [x] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

File to create the test h2 db (at `etc/db/`) from scratch. This will aid anyone who wishes to add more variety to test entities(tables, volumes, functions) available out of the box.

### CODE REVIEW

File: server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+import io.unitycatalog.server.persist.dao.*;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+
+import java.util.*;
+
+import static io.unitycatalog.server.utils.ColumnUtils.*;
+
+public class PopulateTestDatabase {
```
Comment (tdas) [inline server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java:18]: there needs to be instructions one how to run this class

Comment (ravivj-db) [reply server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java:27]: added

Comment (tdas) [reply server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java:27]: how do you run the main method of this class?? Please be more detailed.

Comment (ravivj-db) [reply server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java:27]: Added details for the same.

File: server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+import java.util.*;
+
+import static io.unitycatalog.server.utils.ColumnUtils.*;
+
+/**
+ * This is a utility class to populate the test database with some sample data.
+ * All the quickstart examples in the documentation are based on this sample data.
+ * This class is not part of the main application code and is only used for testing purposes.
+ * To recreate the sample data, first delete the existing database (at /etc/db)
+ * and then run main method of this class by running the command `build/sbt server/populateTestDB`.
```
Comment (tdas) [inline server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java:23]: its `populateTestDatabase` isnt it?

Comment (ravivj-db) [reply server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java:23]: This is just the name of the target specified in the `build.sbt`.

File: server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java [hunk 3]
```diff
@@ -0,0 +1,343 @@
+package io.unitycatalog.server.utils;
```
Comment (tdas) [inline server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java:1]: why did the h2db change?

**Review by tdas — APPROVED** [review]

**Pull request merged**